samples/msal-node-samples/ExpressTestApp/package-lock.json

```diff
@@ -1,2260 +0,0 @@
-{
```

Comment:
You can add an `.npmrc` file with `package-lock=false` to this directory to prevent package-locks from being generated in the future.